src/features/quick-search/__tests__/QuickSearchItem-test.js

```diff
@@ -75,12 +75,24 @@ describe('features/quick-search/QuickSearchItem', () => {
         expect(searchTerm.text()).toEqual('test');
     });
 
+    test('should render Link for item info when shouldNavigateOnItemClick is passed in', () => {
+        const wrapper = mount(<QuickSearchItem itemData={itemData} shouldNavigateOnItemClick />);
+        const itemName = wrapper.find('a.item-name');
+        expect(itemName.prop('href')).toEqual('/file/321');
+    });
+
+    test('should render span for item info when shouldNavigateOnItemClick is not passed in', () => {
+        const wrapper = mount(<QuickSearchItem itemData={itemData} />);
+        const itemName = wrapper.find('.item-name');
+        expect(itemName.prop('href')).toEqual(undefined);
```

Comment:
Would it be possible to add a snapshot in these cases to make sure that a link and span are getting rendered respectively?